### DISCUSSION TITLE
中国科学技术大学

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#4] [University]
> 回答来自群友，我仅负责搬运

**宿舍是上床下桌吗**
是

**教室和宿舍有没有空调**
有

**有独立卫浴吗？没有独立浴室的话，澡堂离宿舍多远**
男生宿舍每层有公共卫浴；中区女生宿舍独立卫浴

**有早自习、晚自习吗**
大一有些班级会安排，强制程度取决于班主任

**有晨跑吗**
没有  \*每年有体测，不及格将导致无法参评部分奖学金/体育只能选体能锻炼课/无法毕业（传闻）

**每学期跑步打卡的要求是多少公里，可以骑车吗**
没有

**寒暑假放多久，每年小学期有多长**
没有 暑假两个月（内含至多三周的小学期），寒假一个月左右

**学校允许点外卖吗，取外卖的地方离宿舍楼多远**
允许外卖，疫情前会送到楼下；现在只能放在校门附近，离宿舍百米左右

**学校附近有地铁站吗**
精 密 仪 器 怕 震（据说以后会有）

**宿舍楼有洗衣机吗**
有

**校园网怎么样**
有，每个月20不限流量
教育网出口有公网 IP
东西区宿舍有网

**每天断电断网吗，几点开始断**
断电时间 23:30 (Sun-Thu), 24:00 (Fri-Sat) 宿舍不通网

**食堂价格贵吗，会吃出异物吗**
取决于是哪间食堂，比较少听说吃出异物（吃到建议索赔）

**洗澡热水供应时间**
全天有，在非高峰时段要放一些冷水才有热水

**校园内可以骑电瓶车吗，电池在哪能充电**
可骑电动车，宿舍无充电区域

**Comment by ilxwhy** [comment]
东区北门不是有一个地铁站吗
